entry_tasks/3_divide_numbers/feat_#3.1_ divide_two_numbers.py

```diff
@@ -0,0 +1,9 @@
+#1
+a = 4
+b = 2
+c = a//b
+print(c)
+
+#2
+v = input(4//2)
```

Comment:
Why it's hardcoded? As a user I want to run your program multiple times and pass my values!